ElfUtils/ElfFile.cpp

```diff
@@ -247,15 +246,18 @@ std::unique_ptr<ElfFile> ElfFile::CreateFromBuffer(std::string_view file_path,
           llvm::object::ObjectFile::createObjectFile(buffer->getMemBufferRef());
 
   if (!object_file_or_error) {
-    return nullptr;
+    return ErrorMessage(
+        absl::StrFormat("Unable to create elf-file \"%s\": %s", file_path,
```

Comment:
Maybe "ELF file" instead of "elf-file" in these messages?

Reply:
done.